validphys2/src/validphys/checks.py

```diff
@@ -71,7 +70,7 @@ def check_can_save_grid(ns, **kwags):
     if not ns['installgrid']:
         return
 
-    write_path = lhapdf.paths()[-1]
+    write_path = lhaindex.get_lha_paths()[-1]
```

Comment:
Or instead of using it, scrap that function everywhere. It doesn't improve readability since it's already very clear and just obfuscates which call is being made to lhapdf. I'm not a huge fan of functions like that tbh

Reply:
scrap lhapath you mean?

(I 100% agree with you on the obfuscation thing, but this might've been born many versions of lhapdf ago)

Reply:
Scrap `get_lha_paths` and `get_lha_datapaths`. 

I genuinely don's see the benefit of having 
```
def get_lha_paths():
    return lhapdf.paths()


def get_lha_datapath():
    return get_lha_paths()[-1]
```
and now that `lhapdf` can be two different things it even hides that behind another layer, meaning I have to jump down to figure out if I'm dealing with `lhapdf_management` or `lhapdf`-classic.

Reply:
Fair. I don't even know why the two exist.
After the review I'll remove all instances and leave only one.